mne/preprocessing/tests/test_ica.py

```diff
@@ -130,6 +130,33 @@ def test_ica_rank_reduction():
 
 
 @requires_sklearn
+def test_ica_reset():
+    """Test ICA resetting"""
+    raw = io.Raw(raw_fname, preload=True).crop(0, stop, False).crop(0.5)
```

Comment:
If we're going to keep this as is, can you make a note (`XXX`) and wherever you copied it from that we should investigate at some point why it matters? It is strange as it should be equivalent to `.crop(0.5, stop, False)`, and I'm worried if it has to be done this way there is some bug we're unaware of...